requirements.txt

```diff
@@ -0,0 +1,3 @@
+cerberus
```

Comment:
Yes, as indicated in the description for this PR, "changes project structure (uses src) to allow imports from within the package". Also required moving `requirements.txt`.

Reply:
Also: it turns out that this is a common problem when developing with `sam`. They provide no guidance about it. It seems that the assumption is that the lambda development will be done as a simple script rather than a module. Obviously if the project is sufficiently complex turning it into a module has advantages. I'm actually pleased that I found a solution to this problem.

Reply:
I'm don't really understand your explanations but I guess i'm fine with it as long as it works.  I was wondering how SAM knows about requirements.txt all since you moved it out of the function folder but I see the change that makes it work now, `CodeUri: .` in temlates.yaml which doesn't see like a good idea to me. Wouldn't bruce's lambda function have it's own requirements.txt file?   where would he put his requirements?

Reply:
@khai I can do another refactor to try to accommodate adding other lambdas. However, I suggest that spending time/money to make a multi-lambda repo should make us take a step back and question whether that's sensible. If I have to do extra work to accommodate another lambda, doesn't it make more sense to keep things isolated and simple?

Reply:
the concept is multiple functions in one lambda NOT multiple lambdas.  Here's a good description, https://github.com/aws/aws-toolkit-jetbrains/issues/1197

I don't think there's any extra work, you just need to put requirements.txt in the right places.

Reply:
This won't work -- the lambdas have two different triggers.